src/Spine.ts

```diff
@@ -419,6 +420,8 @@ export class Spine extends Container implements View
                         );
                     }
 
+                    if (attachment.region.texture.texture !== previousAttachment) this.spineAttachmentsDirty = true;
```

Comment:
would this need a `?` too? `attachment.region?.texture.texture`

Reply:
Sure, I forgot to add it. But is it correct doing just this or is it better to use the batcher `checkAndUpdateTexture` in the `validateRenderable`?

Reply:
oooh you are right! il modify this PR

Reply:
I've just notive that in the piece of code in the message above, I return in the for loop at the first texture change. That's wrong, I should have loop over all the attachments.
But you probably know better than me that :p